packages/@sanity/base/.interopReexport.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
I think this filename should be a private-style `_interopReexport.js`, and not start with `.` so as to not be a hidden file on some systems.